src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,28 @@
+import { useContext, useState } from "react";
+import { ActiveUserContext } from "../contexts/ActiveUserContext";
+import { delCommentById } from "../api";
+
+const DeleteComment = ({author, commentId}) => {
+    const {activeUser} = useContext(ActiveUserContext);
+    const [isDeleted, setIsDeleted] = useState(false);
+    const [isError, setIsError] = useState(false);
+
+    const delhandler = () => {
+        delCommentById(commentId)
+        .then(() => {
+            setIsDeleted(true);
+            setIsError(false);
+        })
+        .catch(err => {
+            setIsError(err);
+        });
+    }
+    
+    if (isDeleted) return <h3>Comment Deleted</h3>
```

Comment:
Personally, I think it would be better to have this message replace the component card entirely, rather than just the delete button. That way it's more obvious that a comment has gone - just a personal preference, though